frontend/src/app/account/history/history.model.ts

```diff
@@ -0,0 +1,40 @@
+export interface MatchingHistory {
+    id: string;
+    collaborator: string; // collaborator username
+    question: string; // question title
+    difficulty: string; // question difficulty
+    topics: string[]; // question topics
+    status: string; // status of the session
```

Comment:
Could use existing enum for difficulty and add a enum for status

Reply:
Done